### PR TITLE
Favicon issue solved

### PR DESCRIPTION
**Pull request by newbazz**
Closes https://github.com/fossasia/2012.fossasia.org/issues/149

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <head>
+    <link rel="shortcut icon" href="">
```
Comment (abishekvashok) [inline _includes/head.html]: pls add a favicon similar to the one used at other summit sites

**Review by abishekvashok — CHANGES_REQUESTED** [review]

File: _includes/head.html (hunk 1 above)
Comment (newbazz) [reply _includes/head.html]: There is no need i think

Comment (newbazz) [reply _includes/head.html]: You mean `<link rel="shortcut icon" href="favicon.ico" type="image/x-icon"/>` this right?

Comment (abishekvashok) [reply _includes/head.html]: yes, pls we need an icon

**Comment by newbazz** [comment]
Changes made

**Review by isuruAb — COMMENTED** [review]
please provide a preview link

**Comment by newbazz** [comment]
@isuruAb preview link as in?

**Comment by abishekvashok** [comment]
https://newbazz.github.io/2012.fossasia.org

**Comment by newbazz** [comment]
Do i need to do anything now in this PR?

File: _includes/head.html [hunk 2]
```diff
@@ -1,4 +1,5 @@
 <head>
+    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon"/>
```
Comment (abishekvashok) [inline _includes/head.html]: Pls add a favicon under the name
`favicon.ico`

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Comment by newbazz** [comment]
@abishekvashok done ;P

**Review by akshatnitd — COMMENTED** [review]
@newbazz I cannot see the changes reflecting in the preview link

**Review by ParthS007 — CHANGES_REQUESTED** [review]
@newbazz  Please update your PR 👍

File: _includes/head.html [hunk 3]
```diff
@@ -1,4 +1,5 @@
 <head>
+    <link rel="shortcut icon" href="favicon.ico" name="favicon.ico" type="image/x-icon"/>
```
Comment (0x48piraj) [inline _includes/head.html:2]: Where's the `ico` file ?

**Comment by abishekvashok** [comment]
I am gonna merge this. Lets not make sure his work was vain. We will release the issues for others to claim!

**Pull request merged**